h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_large.py

```diff
@@ -1,77 +0,0 @@
-from __future__ import print_function
-import sys
-
-sys.path.insert(1, "../../")
-import h2o
-from tests import pyunit_utils
-import os
-
-
-def test_parser_svmlight_column_skip():
-    # generate a big frame with all datatypes and save it to svmlight
-    nrow = 10000
-    ncol = 10
-    seed = 12345
-
-    f1 = h2o.create_frame(rows=nrow, cols=ncol, real_fraction=0.5, integer_fraction=0.5, missing_fraction=0.2,
-                          has_response=False, seed=seed)
-    f2 = h2o.create_frame(rows=nrow, cols=1, real_fraction=1, integer_fraction=0, missing_fraction=0,
-                          has_response=False, seed=seed)
-    f2.set_name(0, "target")
-    f1 = f2.cbind(f1)
-
-    tmpdir = os.path.normpath(os.path.join(os.path.dirname(os.path.realpath('__file__')), "..", "results"))
-    if not (os.path.isdir(tmpdir)):
-        os.mkdir(tmpdir)
-    savefilenamewithpath = os.path.join(tmpdir, 'out.svm')
-    pyunit_utils.write_H2OFrame_2_SVMLight(savefilenamewithpath, f1)  # write h2o frame to svm format
-
-    skip_all = list(range(ncol))
-    skip_even = list(range(0, ncol, 2))
-
-    try:
-        h2o.upload_file(savefilenamewithpath, skipped_columns=skip_all)
-        assert False, "Test should have thrown an exception due to all columns are skipped"  # should have failed here
-    except:
-        pass
-
-    try:
-        h2o.import_file(savefilenamewithpath, skipped_columns=skip_all)
-        assert False, "Test should have thrown an exception due to all columns are skipped"  # should have failed here
-    except:
-        pass
-    try:
-        h2o.import_file(savefilenamewithpath, skipped_columns=skip_even)
-        assert False, "Test should have thrown an exception due to column skipping not supported for svm"  # should have failed here
-
-    except:
-        pass
-
-    # check for correct parsing only
-    checkCorrectSkips(savefilenamewithpath, f1)
```

Comment:
Why are we not checking the correct parsing anymore?

Reply:
@michalk  There are other tests that check for correct parsing.  I think it is okay to skip in this case.  It makes the test run faster.

Reply:
The parsing is cancelled before the job started, there is no result to check, beside check that nothing happened.